server/src/main/resources/server-reference.conf

```diff
@@ -25,6 +25,8 @@ server.downloadsPath = ""
 server.autoDownloadNewChapters = false # if new chapters that have been retrieved should get automatically downloaded
 server.excludeEntryWithUnreadChapters = true # ignore automatic chapter downloads of entries with unread chapters
 server.autoDownloadNewChaptersLimit = 0 # 0 to disable it - how many unread downloaded chapters should be available - if the limit is reached, new chapters won't be downloaded automatically. this limit will also be applied to the auto download of new chapters on an update
+server.autoDownloadExcludeReUploadsFromLimit = false # Decides if re-uploads, that the server wasn't able to handle (chapter number changed - e.g. 105.5 -> 105), should be taking into consideration for the new chapter download limit. e.g. 5 new chapters, 3 re-upload, limit: 2 - disabled = downloads 2 re-uploads / enabled = download all re-uploads + 2 "actual" new chapters
```

Comment:
I feel like we don't need this specific setting, just autoDownloadIgnoreReUploads should work

Reply:
removed